ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/Testnet.hs

```diff
@@ -837,7 +837,7 @@ prop_track_coolingToCold_demotions defaultBearerInfo diffScript =
           notInProgressDemoteToColdForTooLong =
             map (\addr ->
                   Signal.keyedTimeoutTruncated
-                    60
+                    120
```

Comment:
Could you explain why we need to increase it?

Reply:
There was a test failure. I looked into it and it was due to the timeout just barely making it.

Reply:
It isn't